compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -45,7 +45,7 @@ object Completion {
   private def completionMode(path: List[Tree], pos: SourcePosition): Mode = {
     path match {
       case (ref: RefTree) :: _ =>
-        if (ref.name == nme.ERROR) Mode.None
+        if (ref.name == nme.ERROR) Mode.Term | Mode.Type
```

Comment:
@smarter 

> Is this because we don't know if we're completing a term or a type at this point ? The parser should know if we're in term or type context, and could output nme.ERROR or tpnme.ERROR depending on which one it is.

Good idea!

> Also, thinking about it, when completing something in type context we should also complete terms, because a term can have a type member (but a type cannot have a term member)

I've added that too. From the spec, it looks like the path for types needs to include stable members only. I've used `CheckRealizability` for that, but I'm not sure this is correct.